config/locales/de.yml

```diff
@@ -3052,6 +3052,14 @@ de:
         Die Hausaufgabe konnte nicht gelöscht werden. Das kann z.B. daran
         liegen, dass es dafür bereits Abgaben gibt, für die Dateien hochgeladen
         wurden.
+  exception:
+    exception: Exception auf %{host}
+    exception_report: 'Es gab eine Exception auf %{host}:'
+    user_text: >
+      Ups, das hätte nicht passieren sollen. Das ist ein Bug in MaMpf.
+    infos_wanted_html: >
```

Comment:
We already get notified, so maybe tell this the user. and ask for more information or use it as advertisment for programmers.

Reply:
I reformulated it. The contact infos are in the footer anyway.